java/code/src/com/suse/manager/webui/services/SaltServerActionService.java

```diff
@@ -987,7 +986,7 @@ private List<SaltState> convertToState(long actionChainId, ServerAction serverAc
                         }
                         else {
                             return new SaltModuleRun(stateId,
-                                    "state.apply",
+                                    minion.isTransactionalUpdate() ? "transactional_update.apply" : "state.apply",
```

Comment:
Why do we need to do it explicitly? Transactional executor should translate state.apply to transactional_update.apply automatically.